suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -420,7 +420,7 @@ export const networks = {
     sol: {
         name: 'Solana',
         networkType: 'solana',
-        bip43Path: "m/44'/501'/i'/0'", // TODO(vl): accounts and paths
+        bip43Path: "m/44'/501'/i'", // TODO(vl): decide on the default path
```

Comment:
I chose `bip44` path as the default, because it seems more inline with the other coins. If you agree I will remove the comment and consider this final

Reply:
cc @matejcik

Reply:
If i'm reading it right, Ledger Live also does the 3-component path? Let's join their voice of reason in this cursed landscape.

Reply:
Don’t know, wondering if it is necessary to support all 3 types of accounts (in Suite) 🤔

Reply:
According to [this](https://help.phantom.app/hc/en-us/articles/12988493966227-What-derivation-paths-does-Phantom-wallet-support-), they are using multiple paths for solana. 

If I understand that correctly, there is only one root account? What is the usecase of this root and change accounts? @PeterBenc

Reply:
Yes, bip44Root path type supports only a single account, since it does not have accountIndex, it's just `m/44'/501'`

Regarding use cases, there are other types of derivation paths that are/were used by other wallets, so there might be some users who could recover such accounts in the suite. 

It really depends on whether you want to support more types of accounts. Since the UI is IMO not very much suited for it, and the scenario of users recovering previously created accounts in the suite is IMO not that common, it might make sense to support just one type and add others if users ask for it :thinking: On the other hand, supporting other types is free and already implemented.